tensorflow_graphics/projects/pointnet/augment.py

```diff
@@ -0,0 +1,33 @@
+#Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Data augmentation utility functions."""
+import numpy as np
+import tensorflow as tf
+from tensorflow_graphics.geometry.transformation import rotation_matrix_3d
+
+def jitter(points, stddev=0.01, clip=0.05):
+  """Randomly jitters points with (clipped) Gaussian noise."""
```

Comment:
could be nice to document the parameters of all the functions similar to the rest of TFG.

Reply:
I was thinking to move these to a centralized home inside tfg/core later.
Will add dense documentation when that happens, sounds good?